routes/embeddingsRoutes.js

```diff
@@ -76,6 +76,8 @@ router.post('/', async (req, res) => {
 		? req.body.input.join(' ')
 		: req.body.input;
 	const scriptArgs = ['-m', modelPath, '-p', input.replace(/"/g, '\\"')];
+	const promptTokens = Math.ceil(input.length / 4);
+	let completionTokens = 0;
```

Comment:
looking at the official api: https://platform.openai.com/docs/api-reference/embeddings/create#embeddings/create-model

doesn't appear that we need completion tokens here (also doesn't quite make sense), can you please remove completion token?

Reply:
I've emoved it and updated.